docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2916 @@
+* Adds the ability to edit data views in the *Lens* flyout {kibana-pull}142362[#142362]
+* Adds conditional operations in the *Lens* Formula {kibana-pull}142325[#142325]
+* Adds the ability to explore fields in Discover from *Lens* {kibana-pull}142199[#142199]
+* Adds the ability to open *TSVB* Gauge visualizations in *Lens* {kibana-pull}142187[#142187]
+* Adds new defaults function in *Lens* {kibana-pull}142087[#142087]
+* Adds support for mustache context variables with periods {kibana-pull}143703[#143703]
+* Adds explore matching indices to data view menu {kibana-pull}141807[#141807]
+* Adds control in the *Lens* annotations layer menu for global filters {kibana-pull}141615[#141615]
+* Adds field filter to popover in *Lens* {kibana-pull}141582[#141582]
+* Improves the performance for large formulas in *Lens* {kibana-pull}141456[#141456]
+* Improves the Quickfunction in-product assistance in *Lens* {kibana-pull}141399[#141399]
+* Adds bit formatter in *Lens* {kibana-pull}141372[#141372]
+* Adds the ability to open aggregation-based pie visualizations in *Lens* {kibana-pull}140879[#140879]
+* Adds the ability to open *TSVB* metric visualizations in *Lens* {kibana-pull}140878[#140878]
+* Adds the ability to open aggregation-based table visualizations in *Lens* {kibana-pull}140791[#140791]
+
+Machine Learning::
+* Data Frame Analytics: Highlight filtered data in scatterplot charts {kibana-pull}144871[#144871]
+* Allow updates for number of allocations and priority for trained model deployments {kibana-pull}144704[#144704]
+* Switch from normal sampling to random sampler for Index data visualizer table {kibana-pull}144646[#144646]
+* Explain Log Rate Spikes: Replace chunks of queries with concurrent queue {kibana-pull}144220[#144220]
+* Explain Log Rate Spikes: Allow to continue failed stream {kibana-pull}143301[#143301]
+* Entity filter for the Notifications page {kibana-pull}142778[#142778]
+* Show an info callout for new notifications {kibana-pull}142245[#142245]
+* Adding dashboard custom url to lens created jobs {kibana-pull}142139[#142139]
+* Adds ML open API output to appendix {kibana-pull}141556[#141556]
+
+Management::
+Adds missing geo aggs to autocomplete in Console {kibana-pull}141504[#141504]
+
+Maps::
+* Adds the ability to invert color ramp and size {kibana-pull}143307[#143307]
+* Adds layer groups {kibana-pull}142528[#142528]
+* Adds the ability to hide or show all layers {kibana-pull}141495[#141495]
+
+Observability::
+* Adds kibana.alert.time_range field to Alert-As-Data mappings and populate it {kibana-pull}141309[#141309]
+* Alert summary widget new design {kibana-pull}141236[#141236]
+* Adds histogram support for avg, max, min, sum and percentiles {kibana-pull}139770[#139770]
+
+Platform::
+* Allow date functions in formula {kibana-pull}143632[#143632]
+* Adds option to disable cursor sync on dashboards {kibana-pull}143355[#143355]
```

Comment:
Both of these probably should be categorized under visualizations instead of platform. Or maybe the first under Lens & the second under Dashboards?

cc @stratoula

Reply:
Yes the first is about Lens and the second one is about dashboards (but refers only to visualizations panels (not maps or saved searches) as Luke correctly says!